test/parallel/test-tls-canonical-ip.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+require('../common');
+
+// Test conversion of IP addresses to the format returned
+// for addresses in Subject Alternative Name section
+// of a TLS certificate
+
+const assert = require('assert');
+const canonicalIP = process.binding('cares_wrap').canonicalIP;
+
+assert.strictEqual(canonicalIP('127.0.0.1'), '127.0.0.1');
+assert.strictEqual(canonicalIP('10.1.0.1'), '10.1.0.1');
+assert.strictEqual(canonicalIP('::1'), '::1');
+assert.strictEqual(canonicalIP('fe80:0:0:0:0:0:0:1'), 'fe80::1');
+assert.strictEqual(canonicalIP('fe80:0:0:0:0:0:0:0'), 'fe80::');
+assert.strictEqual(
+  canonicalIP('fe80::0000:0010:0001'),
+  'fe80::10:1');
```

Comment:
This fits on one line.  If it didn't, house style is to write it like this:
```js
assert.strictEqual(canonicalIP('fe80::0000:0010:0001'),
                   'fe80::10:1');
```
Only if that wouldn't fit in 80 columns would you use one-argument-one-line style but then please indent by four spaces.  (Truth be told, I would have expected the linter to complain about that.)